### ISSUE TITLE
voiceover accessibility issue with tasks and habits

### ISSUE THREAD

**Issue by rilygreat** [HabitRPG/habitrpg-ios#1236]
> The following lines help us find and squash the Bug you encountered. Please do not delete/change them.
> iOS Version: 16.0
> Device: iPhone 12 Pro
> App Version: 3.5.5 (695)
> User ID: 8c3906cd-5983-4a09-ac6f-602c19fbb11c
> Level: 2
> Class: warrior
> Is in Inn: false
> Uses Costume: false
> Custom Day Start: 0
> 
> 
> Sent from my iPhone
> Hi
> I am blind so I use voiceover but there are some accessibility issues with the app. For example, selecting the difficulty of a task and setting a habit as positive or negative is not accessible now because the buttons aren’t labeled correctly.

**Comment by EricMiddelhove** [comment]
I'd like to work on this issue :)

**Comment by saraolson** [comment]
@EricMiddelhove great, i'll mark it as in progress for you! just let us know if you need to drop it for any reason

**Comment by EricMiddelhove** [comment]
Hi @saraolson pull request is already out, and to my knowledge already merged to develop ☺️👍🏽

**Comment by saraolson** [comment]
oh awesome, then i'll be testing it in the beta build later today then adding it to your habitica profile after that, ty!
